### PR TITLE
Added Contributing.md

### PR DESCRIPTION
**Pull request by dohro-github**
added the changes @quellus wanted

### CODE REVIEW

File: wiki/Contributing.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+Your pull request will be reviewed by one of our team members. We may ask for changes or clarifications. Please be responsive and address any feedback as soon as possible.
+
+### Aditional Ntoes
+
+- Adhere to the existing code style and conventions.
+- Write clear, concise commit messages.
+- Include comments and documentation as necessary.
+- Ensure your changes do not break existing functionality.
+
+### Guidlines
```
Comment (quellus) [inline wiki/Contributing.md:82]:
```suggestion
### Guidelines
```

File: wiki/Contributing.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+4. Select the branch you pushed your changes to in your forked repository.
+5. Click 'Create pull request' and fill out the required information.
+
+## Insight
+
+### Code Review
+
+Your pull request will be reviewed by one of our team members. We may ask for changes or clarifications. Please be responsive and address any feedback as soon as possible.
+
+### Aditional Ntoes
```
Comment (quellus) [inline wiki/Contributing.md:75]:
```suggestion
### Additional Notes
```

File: wiki/Contributing.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+## How to contribute
+
+### Getting Started
+
+1. Fork the repository: Click the 'Fork' button at the top right corner of the repository page.
+
+2. Clone your forked repository:
+
+> git clone https://github.com/quellus/GDTuber.git
```
Comment (quellus) [inline wiki/Contributing.md:26]: For all of these block quotes, please change them to the code block format we have in the README

 ```sh
git clone https://github.com/quellus/GDTuber.git

File: wiki/Contributing.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+> git push origin feature-or-bugfix-name
+
+2. Create a Pull Request:
+
+1. Go to the [GDTuber](https://github.com/quellus/GDTuber/) repository.
+2. Click on the 'Pull requests' tab.
+3. Click the 'New pull request' button.
+4. Select the branch you pushed your changes to in your forked repository.
+5. Click 'Create pull request' and fill out the required information.
```
Comment (quellus) [inline wiki/Contributing.md:67]: The numbering gets a little weird with this one. I think we need to tab these over to make it work right

```suggestion
  1. Go to the [GDTuber](https://github.com/quellus/GDTuber/) repository.
  2. Click on the 'Pull requests' tab.
  3. Click the 'New pull request' button.
  4. Select the branch you pushed your changes to in your forked repository.
  5. Click 'Create pull request' and fill out the required information.
```

File: wiki/Contributing.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+- Adhere to the existing code style and conventions.
+- Write clear, concise commit messages.
+- Include comments and documentation as necessary.
+- Ensure your changes do not break existing functionality.
+
+### Guidlines
+
+- Be respectful and considerate of others.
+- Provide constructive feedback.
+- Keep discussions focused and on topic.
```
Comment (quellus) [inline wiki/Contributing.md:86]:
```suggestion
- Keep discussions focused and on topic.

Thank you for your contributions!

For any questions or additional help, feel free to open an issue or contact us on our [Discord server](https://discord.com/invite/RzaVWjPU8s).
```

**Review by quellus — CHANGES_REQUESTED** [review]
You shouldn't have to make a new Pull Request each time. If you add new commits to the same branch, it will update this Pull Request

**Comment by quellus** [comment]
Closes #104

**Comment by quellus** [comment]
@dohro-github Is there any way we can help get this merged?

**Comment by dohro-github** [comment]
> @dohro-github Is there any way we can help get this merged?

I think you can just fix the errors in my md code then merge it, if that is what you are asking

**Pull request merged**